pyface/i_widget.py

```diff
@@ -73,7 +73,7 @@ def has_focus(self):
             Whether or not the widget has keyboard focus.
         """
 
-    def create(self):
+    def create(self, parent=None):
```

Comment:
Question: did you consider making the `parent` argument keyword-only? I think best practices should be to always provide the parent by keyword, but I'm not sure whether it's viable to enforce that at the level of the signature.

Reply:
No, it seems more common to provide parents without keywords in the `__init__` methods, so I wanted to keep that similarity, and in the example code I found that a lot of the usage would have been of the form `foo_widget.create(parent=parent)` in which case `foo_widget.create(parent)` is I think better.

Reply:
Okay. I personally find `foo_widget.create(parent)` more confusing, and would prefer to see the keyword argument everywhere.